### PR TITLE
Align aeminer dependency to latest one

### PR DESCRIPTION
**Pull request by lucafavatella**
Delivers https://www.pivotaltracker.com/story/show/164984716

TODO:
* [x] Update commit of dependency once https://github.com/aeternity/aeminer/pull/10 merged.

### CODE REVIEW

**Comment by lucafavatella** [comment]
Updated following merge of https://github.com/aeternity/aeminer/pull/10

File: apps/aecore/src/aec_conductor.erl [hunk 1]
```diff
@@ -86,6 +86,7 @@
 -define(SERVER, ?MODULE).
 
 -define(DEFAULT_MINING_ATTEMPT_TIMEOUT, 60 * 60 * 1000). %% milliseconds
+-define(NONCE_RANGE, 1000000000000000000000000). %% 10^24, approx. 2^80
```
Comment (hanssv) [inline apps/aecore/src/aec_conductor.erl]: Why use a constant that is bigger than the max allowed nonce?

Comment (lucafavatella) [reply apps/aecore/src/aec_conductor.erl]: You are correct. I corrected force pushing new commit. Thanks.

**Comment by lucafavatella** [comment]
I see failure in [CI job `test_roma`](https://circleci.com/gh/aeternity/aeternity/17863#tests/containers/1):
```
%%% aecore_txs_SUITE: .Makefile:192: recipe for target 'ct-roma' failed
make: *** [ct-roma] Terminated
Too long with no output (exceeded 10m0s)
```

**Pull request merged**